caom2-search-server/examples/default/src/intTest/java/ca/nrc/cadc/search/integration/CAOMSearchBrowserTest.java

```diff
@@ -31,33 +31,32 @@
  ****  C A N A D I A N   A S T R O N O M Y   D A T A   C E N T R E  *****
  ************************************************************************
  */
+
 package ca.nrc.cadc.search.integration;
 
 
 import org.junit.Test;
 import org.openqa.selenium.By;
 
 
-public class CAOMSearchBrowserTest extends AbstractAdvancedSearchIntegrationTest
-{
+public class CAOMSearchBrowserTest extends AbstractAdvancedSearchIntegrationTest {
     private static final By ONE_CLICK_DOWNLOAD_LINK_ROW_3_ID_BY = By.id("_one-click_vov_3");
 
 
-    public CAOMSearchBrowserTest() throws Exception
-    {
+    public CAOMSearchBrowserTest() throws Exception {
         super();
     }
 
 
     @Test
-    public void searchCAOM() throws Exception
-    {
+    public void searchCAOM() throws Exception {
         CAOMSearchFormPage searchFormPage = goTo(endpoint, null, CAOMSearchFormPage.class);
 
         searchFormPage.enterObservationID("692512");
         searchFormPage.enterValidTarget("210.05  54.3");
 
         searchFormPage.reset();
+        searchFormPage.uncheckMAQ();
```

Comment:
Because reset() will reset the switch to where it was when the page loaded (i.e. default setting).  This is the original browser test, which relies on the switch being unchecked.